src/Idris/IBC.hs

```diff
@@ -517,13 +511,15 @@ processImports reexp phase fname ar = do
         ibcsd <- valIBCSubDir i
         ids <- rankedImportDirs fname
         putIState (i { imported = f : imported i })
-        let phase' = case phase of
-                         IBC_REPL _ -> IBC_REPL False
-                         p -> p
+        let (phase', reexp') = case phase of
```

Comment:
Just a small note on style, I prefer not to have a dangling `case`.
Would it possible to use the following instead?

```haskell
let (phase', reexp') = 
    case phase of
        IBC_REPL ....
```

Reply:
No worries. I prefer that style too. I used `hindent` to help me format it like that (but I cherry picked the expression and then paste it back into place so as not to reformat the rest of the definition). A lot of the Idris code follows a different style though 😄 .